Source/DafnyCore/Rewriters/InductionRewriter.cs

```diff
@@ -156,52 +165,149 @@ void ComputeInductionVariables<VarType>(IToken tok, List<VarType> boundVars, Lis
         return;
       }
 
-      // The argument list was legal, so let's use it for the _induction attribute
+      // The argument list was legal, so let's use it for the _induction attribute.
+      // Next, look for matching patterns for the induction hypothesis.
+      if (lemma != null) {
+        var triggers = ComputeAndReportInductionTriggers(lemma, ref attributes, goodArguments, body);
+        if (triggers.Count == 0) {
+          var suppressWarnings = Attributes.Contains(attributes, "nowarn");
+          var warningLevel = suppressWarnings ? ErrorLevel.Info : ErrorLevel.Warning;
+
+          Reporter.Message(MessageSource.Rewriter, warningLevel, null, tok,
+            $"Could not find a trigger for the induction hypothesis. Without a trigger, this may cause brittle verification. " +
+            $"Change or remove the {{:induction}} attribute to generate a different induction hypothesis, or add {{:nowarn}} to silence this warning. " +
```

Comment:
Is there a way to make it produce a quantifier without triggers (and thus recover the previous behavior?)